src/orient/rational-calc.orient

```diff
@@ -0,0 +1,354 @@
+
+  SNARK:
+    describe(inclusion_proof, "Constraints for checking an inclusion proof in a binary merkle tree as large as the sector")
+    describe(post_inclusion_proofs, "Constraints for checking all inclusion proofs (in PoSt)")
+    describe(ticket_proofs, "Constraints for checking the partial ticket (in PoSt)")
+    describe(ticket_constraints, "Constraints of a partial ticket per challenged node (in PoSt)")
+    describe(post_snark_constraints, "Constraints for a PoSt")
+    describe(post_snark_partitions, "Number of partitions of a PoSt proof")
+    describe(post_snark_size, "Size of a PoSt proof (bytes)")
+    describe(post_snark_size_kb, "Size of a Post proof (KiB)")
+    describe(post_snark_partition_constraints, "Number of constraints for a PoSt partition")
+
+    inclusion_proof = merkle_tree_hash_constraints * log2(nodes)
+    post_inclusion_proofs = inclusion_proof * post_challenges
+    ticket_proofs = ticket_constraints * post_challenges
+
+    post_snark_constraints = ticket_proofs + post_inclusion_proofs
+    post_snark_partitions = post_snark_constraints / post_snark_partition_constraints
+    post_snark_size = post_snark_partitions * snark_size
+    post_snark_size_kb = post_snark_size / 1024
+
+ProofOfReplication:
+  Security:
+    describe(porep_challenges, "Number of challenges in a PoRep proof")
+    describe(porep_lambda, "Bits of security of a PoRep proof")
+    describe(spacegap, "Maximum difference in storage between an honest prover and a maliciois prover")
+
+    porep_challenges = -porep_lambda / (log2(1-spacegap/4))
+
+  Graph:
+    describe(nodes, "Number of nodes in a sector")
+    describe(sector_size, "Size of a sector (bytes)")
+    describe(sector_size_gb, "Size of a sector (GiB)")
+
+    describe(window_nodes, "Number of nodes in a window")
+    describe(window_size, "Size of a window (bytes)")
+    describe(window_size_mb, "Size of a window (MiB)")
+    describe(windows, "Number of windows in a sector")
+
+    describe(graph_parents, "Number of parents of a node in the PoRep window graph")
+    describe(drg_parents, "Number of DRG parents of a node in the PoRep window graph")
+    describe(expander_parents, "Number of Chung Expander parents of a node in the PoRep window graph")
+    describe(drg_layers, "Number of layers in the DRG")
+
+    describe(node_size, "Size of a node (bytes)")
+    describe(node_size_gb, "Size of a node (GiB)")
+    describe(kdf_hash_size, "Size of the input to the KDF hash (bytes)")
+    describe(kdf_hash_size_gb, "Size of the input to the KDF hash (GiB)")
+
+    sector_size = sector_size_gb * b_in_gb
+    graph_parents = drg_parents + expander_parents
+    nodes = sector_size / node_size 
+    node_size_gb = node_size / b_in_gb 
+    kdf_hash_size_gb = graph_parents * node_size_gb
+    kdf_hash_size = kdf_hash_size_gb * b_in_gb
+
+    window_size = window_size_mb * 1024 * 1024
```

Comment:
You might as well create a `b_in_mb` utility constant too, for consistency and to minimize the points where an auditor has to re-verify that the right factors were provided.

Reply:
fixed in latest